default/stringtables/en.txt

```diff
@@ -1891,6 +1891,9 @@ Enable autosave when resigning or closing the game.
 OPTIONS_DB_AUTOSAVE_HOSTLESS
 Enable autosave in hostless mode.
 
+OPTIONS_DB_AUTOSAVE_HOSTLESS_TIMEOUT
+Seconds when autosave happened in hostless mode. 0 if disabled.
```

Comment:
meaning unclear

Reply:
why only in hostless mode?

Reply:
would also be good to have an automatic turn timer, though that's a separate issue

Reply:
Tried to rephrase it.

Reply:
The host can manually save game. I could remove this restriction although.

Reply:
Now timed autosave works in any mode.

Reply:
Turn timer requires some time sync with client so player would know correct time before turn change.